install_config/install/deploy_router.adoc

```diff
@@ -99,6 +99,15 @@ communication between OpenShift components is secured by TLS and uses various
 certificates and authentication methods. Use the `--credentials` option to
 specify what credentials the router should use to contact the master.
 
+[IMPORTANT]
+====
+Routers directly attach to port 80 and 443 on all interfaces on a host. Restrict
+routers to hosts where port 80/443 is available and not being consumed by
+another service, and set this using node selectors and the
+link:admin_guide/scheduler.html[scheduler configuration]. As an example, you can
+achieve this with a dedicated router or infrastructure nodes.
```

Comment:
I think it should be "with dedicated router or infrastructure nodes", not "a dedicated router". @thoraxe ack?